go.mod

```diff
@@ -25,6 +25,7 @@ require (
 	golang.org/x/xerrors v0.0.0-20220411194840-2f41105eb62f // indirect
 	google.golang.org/grpc v1.40.0
 	gopkg.in/yaml.v2 v2.4.0
+	honnef.co/go/tools v0.2.1 // indirect
```

Comment:
how we add this vendor here?

Reply:
try to run those and see if still nothing changed.
```
go mod tidy
go mod vendor
```

Reply:
trying to remove it and see